### PR TITLE
[#61] Add script for setting bitcoin related params.

### PR DESCRIPTION
**Pull request by pafff**

### CODE REVIEW

File: bin/set_bitcoin_related_network_params.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    'class CBitcoinAddress.*?'
+    'PUBKEY_ADDRESS_TEST = (.*?),.*?'
+    'SCRIPT_ADDRESS_TEST = (.*?),',
+    re.S
+)
+
+
+def add_param_lines(lines, index, testnet=False):
+    result = set_params(lines, index, testnet)
+    if result:
```
Comment (szerepak) [inline bin/set_bitcoin_related_network_params.py]: `return result if result else lines`

File: bin/set_bitcoin_related_network_params.py [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    port_line_offset = next((i for i, line in enumerate(lines[index:]) if 'port =' in line), None)
+
+    if github_line_offset is None or port_line_offset is None:
+        return
+
+    port_line_index = index + port_line_offset
+    port_line = lines[port_line_index]
+    github_line = lines[index + github_line_offset]
+
+    if len(lines) == port_line_index+1 or not lines[port_line_index + 1].strip():
```
Comment (szerepak) [inline bin/set_bitcoin_related_network_params.py]: `port_line_index + 1` (spaces between)

**Review by szerepak — APPROVED** [review]
Looks cool, only minor comments.

**Review by trojkat — APPROVED** [review]
`chmod +x bin/set_bitcoin_related_network_params.py`

**Pull request merged**